src/diagram/components/quantity-node.tsx

```diff
@@ -1,12 +1,15 @@
+import classnames from "classnames";
```

Comment:
```suggestion
import classNames from "classnames";
```
We generally camel-case the import